### PR TITLE
Week5 day1

### PR DESCRIPTION
**Pull request by MoMenne**
Adding in the walkthrough for Week5 Day1

### CODE REVIEW

**Comment by MoMenne** [comment]
Merging in changes for week5-day1

File: docs/_sources/week02/spring-application/index.rst.txt [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+.. _week2_spring-application:
+
+=============================================================
+Project Week2 Setup: Spring, Git, GitLab, PostGIS, & IntelliJ
+=============================================================
+
+Before we can begin working on the requirements for this week, we will need to first setup the tools we will be using throughout this week.
+
+Tasks:
```
Comment (welzie) [inline docs/_sources/week02/spring-application/index.rst.txt:11]: This list of tasks is a duplicate of the index on the left side of the page, but it doesn't really matter to me if you lave it or not.

File: docs/_sources/week02/spring-application/index.rst.txt [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+    - Project Metadata Description: *CDC Zika Dashboard*
+    - Project Metadata Package Name: *org.launchcode.zikaDashboard*
+    - Project Metadata Packaging: *Jar*
+    - Project Metadata Java Version: *8*
+    - Dependencies: *Web*
+    - Dependencies: *Thymeleaf*
+    - Dependencies: *JPA*
+    - Dependencies: *DevTools*
+
+Double check that you have filled out the informatoin correctly, and then click Generate Project. This will prompt your browser to download a file called zikaDashboard.zip, go ahead and download it, and take notice of where it is downloaded. Most machines default to a Download directory off of your Home directory.
```
Comment (welzie) [inline docs/_sources/week02/spring-application/index.rst.txt]: informatoin typo

File: docs/_sources/week02/spring-application/index.rst.txt [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+We will want to move the zikaDashboard.zip file to wherever you typically store your projects, and then unzip the file.
+
+You command will likely be different, but I will move my zikaDashboard.zip file to my projects folder with the following command.
```
Comment (welzie) [inline docs/_sources/week02/spring-application/index.rst.txt]: I don't know if writing styles matter, but most of the other content does not use "I" statements.  Maybe change this to "One example of how to do this is"

**Pull request merged**